docs/antora/modules/ROOT/pages/Configuration.adoc

```diff
@@ -501,35 +501,6 @@ consistent configuration: every module often has the same scala-version, uses
 the same testing framework, etc. and all that can be extracted out into the
 `trait`.
 
-== Global configuration
```

Comment:
Oh, I think users need this to get custom repositories to work when behind a firewall.

Reply:
The replacement is probably using Coursier's custom repositories environment variable https://get-coursier.io/docs/other-repositories, but I haven't used it before myself so not sure.

Reply:
I think this does not work when authentication is needed. But I'm not 100% sure.

Reply:
We'd need to put the functionality into `MillBuildBootstrapModule` then, since it'll need to take effect before we download mill-scala-worker to compile `build.sc`. Maybe a config file or system prop or env var or something.

Reply:
Did you already remove the respective `--predef` and `--no-default-predef` cli options?

Reply:
Or we could just bundle the jars used by `MillBuildBootstrapModule` (compiler bridge, worker classpath) with the mill executable. They always need to get downloaded anyway, so this would save us round trips and simplify hermetic builds: you can always compile and run a `build.sc` without network calls, and then you could use your `mill-build/build.sc` to configure repositories for the main `build.sc` to run

Reply:
I had the same thought. By bundling, you mean we either embed or extract some local mini-repository, such that we don't need to special handle that case in the code?